src/style.css

```
@@ -1 +1,23 @@
 /* styles go here */
+.field {
+  display: block;
+  margin: 10px 0;
```

Лучше указывать с одной стороны отступы
```suggestion
  margin-bottom: 20px;
```